### PR TITLE
Give more time before activating BM capping algo change

### PR DESCRIPTION
**Pull request by ghost**
Related to #6949.  Bisq-network/proposals/issues/412

All bisq clients must be upgraded before the activation date.

If one partner to a trade is running a prior release, trade will fail due to the delayed payout transaction not matching.

Change burning man algorithm activation date `Jan 1 2024` -> `Feb 1 2024`.

### CODE REVIEW

**Review by alejandrogarcia83 — APPROVED** [review]
utACK

**Comment by ghost** [comment]
The Bisq1 1.9.15 release has been delayed longer than anyone anticipated.  I (and presumably @stejbac) thought it would happen Dec 2023.  It did not, and this PR was created to extend the [proposal 412](https://github.com/bisq-network/proposals/issues/412#issuecomment-1492851657) activation date into the future.  March 2024 we're still waiting for release, and the prop 412 activation date needs to be pushed forward again into the future.  I [raised this point to the maintainers](https://github.com/bisq-network/bisq/pull/7035#pullrequestreview-1878202028) but it has [not been addressed](https://github.com/bisq-network/bisq/blob/117f282a5097cc45c9319b3a45420120219ea11f/core/src/main/java/bisq/core/dao/burningman/DelayedPayoutTxReceiverService.java#L59C5-L60C119).



**Leaving the feature activated by default puts users who build from source [at risk of failed trades](https://github.com/bisq-network/proposals/issues/412#issuecomment-1938993395).**

I will create another PR to push the activation date forward another two months.

@alejandrogarcia83

**Pull request merged**
